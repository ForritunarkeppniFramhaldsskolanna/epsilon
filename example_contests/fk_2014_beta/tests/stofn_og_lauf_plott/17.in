83
55 92 89 65 72 73 33 32 27 69 96 54 98 51 61 96 22 67 46 39 91 58 58 78 71 19 35 19 78 86 51 61 49 49 15 18 60 11 19 36 58 37 24 19 63 91 76 50 55 20 37 92 44 20 83 23 72 49 61 35 31 59 85 36 55 49 11 94 53 86 70 45 65 29 84 28 84 49 42 40 79 44 75
