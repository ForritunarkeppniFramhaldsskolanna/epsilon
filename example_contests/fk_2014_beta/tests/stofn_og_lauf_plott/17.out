1 11589999
2 00234789
3 1235566779
4 02445699999
5 011345558889
6 011135579
7 0122356889
8 3445669
9 11224668
